sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableServiceAsyncClient.java

```diff
@@ -142,6 +144,15 @@ ClientLogger getLogger() {
         return this.logger;
     }
 
+    /**
+     * Gets this client's {@link SerializerAdapter}.
+     *
+     * @return This client's {@link SerializerAdapter}.
+     */
+    SerializerAdapter getSerializerAdapter() {
```

Comment:
Why is this required? This must be an implementation detail and shouldn't be used in tests.